script/request_order.ts

```diff
@@ -1,5 +1,12 @@
-import "dotenv/config";
+require('dotenv').config({ path: '../.env' });
```

Comment:
Execute from base directory

Reply:
I had issue with my environment , I wasn't able to find PRIVATE_KEY and .env  even when I load it that why I use path repointing